### PR TITLE
Fix for IE

### PR DESCRIPTION
**Pull request by nastikue**
## Motivation

https://qna.askco19.com/ doesn't work in IE 

image

## Implementation

update(index, package.json): add libs and polyphills for IE

## Screenshots/Links

After libraries and polyfills implementation no IE specific errors in console:

image

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -26,7 +26,10 @@
     <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet' async="true">
     <link href='https://fonts.googleapis.com/css?family=Raleway' rel='stylesheet' async="true">
 
-     <script src="https://cdn.botframework.com/botframework-webchat/latest/webchat.js"></script>
+    <script src="//cdn.jsdelivr.net/bluebird/3.5.0/bluebird.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/fetch/2.0.3/fetch.js"></script>
+    <script src="https://cdn.polyfill.io/v2/polyfill.min.js"></script>
+    <script src="https://cdn.botframework.com/botframework-webchat/latest/webchat.js"></script>
```
Comment (aviskarkc10) [inline client/public/index.html:32]: @nastikue i think we should load these asynchronously by adding `defer`
https://flaviocopes.com/javascript-async-defer/

Comment (nastikue) [reply client/public/index.html:32]: @aviskarkc10 a good point, will fix it asap

Comment (nastikue) [reply client/public/index.html]: > @nastikue i think we should load these asynchronously by adding `defer`
> https://flaviocopes.com/javascript-async-defer/

Fix done, both defer and async props were added as defer doesn't work on IE properly

**Comment by aviskarkc10** [comment]
awesome work @nastikue

**Pull request merged**